### PR TITLE
Add config and update spaces to use SLEVE coords (optional) with topography

### PR DESCRIPTION
**Pull request by akshaysridhar**

### CODE REVIEW

**Comment by akshaysridhar** [comment]
https://buildkite.com/clima/climaatmos-gpulongruns/builds/72 (Buildkite run corresponding to parameters in PR)

File: config/longrun_configs/longrun_aquaplanet_rhoe_equil_55km_nz63_clearsky_0M_earth_sleve.yml
Comment (szy21) [inline config/longrun_configs/longrun_aquaplanet_rhoe_equil_55km_nz63_clearsky_0M_earth_sleve.yml:1]: Should we set `mesh_warp_type` to SLEVE for this config?

**Review by szy21 — APPROVED** [review]
Thanks!

File: config/longrun_configs/longrun_aquaplanet_rhoe_equil_55km_nz63_clearsky_0M_earth_sleve.yml
Comment (akshaysridhar) [reply config/longrun_configs/longrun_aquaplanet_rhoe_equil_55km_nz63_clearsky_0M_earth_sleve.yml:1]: Good catch - I had hardcoded the common spaces while testing the gravity wave test case. Fixed.

**Pull request merged**